packages/next-app/src/pages/index.tsx

```diff
@@ -33,12 +35,32 @@ const Home: NextPage = () => {
   }, []);
 
   return (
-    <Box>
+    <Box m="0" w="100vw" h="100vh" background="blue">
       <Head>
         <title>$CODE Claim Page</title>
       </Head>
-      <Button label="TODO" />
-      <Container>{claimPeriodEnds}</Container>
+      <Flex direction="row" flexWrap="wrap">
+        <Box
+          w={["100vw", "50vw"]}
```

Comment:
```suggestion
          w={{ base: "100vw", lg: "50vw" }}
```
I think a breakpoint like this works a bit better for our use case. Otherwise the claim card will have to shrink too much to still fit.

Reply:
@embiem did adjust in claim card PR. either this one could be merged or I just close this PR later and push everything with the claim card. whatever you prefer.